src/components/core/scrollbars/scrollbars.tsx

```diff
@@ -0,0 +1,138 @@
+import type * as Stitches from '@stitches/react';
+import React from 'react';
+import { styled } from 'Styles/stitches.config';
+
+const ScrollbarsContainer = styled('div', {
+    overflowY: 'auto',
+    overflowX: 'hidden',
+    maxHeight: '100%',
+    marginRight: '-14px',
+    paddingRight: '10px',
+    scrollBehavior: 'smooth',
+
+    '&::-webkit-scrollbar': {
+        width: '4px',
+        height: '4px',
+        backgroundColor: 'transparent',
+    },
+    '&::-webkit-scrollbar-thumb': {
+        borderRadius: '4px',
+        backgroundColor: '#D6DADB',
+        opacity: '0.16',
+    },
+
+    variants: {
+        dark: {
+            true: {
+                '&::-webkit-scrollbar-thumb': {
+                    backgroundColor: '#333333',
+                },
+            },
+        },
+        autohide: {
+            true: {
+                '&::-webkit-scrollbar-thumb': {
+                    display: 'none',
+
+                    '&:hover': {
+                        display: 'unset',
+                    },
+                },
+            },
+        },
+        has_horizontal: {
+            true: {
+                overflowX: 'auto',
+            },
+        },
+        is_scrollbar_hidden: {
+            true: {
+                marginRight: '0',
+                paddingRight: '0',
+                '&::-webkit-scrollbar': {
+                    display: 'none',
+                },
+            },
+        },
+        is_only_horizontal: {
+            true: {
+                marginRight: '0',
+                paddingRight: '0',
+                overflowY: 'hidden',
+                overflowX: 'auto',
+            },
+        },
+        is_only_horizontal_overlay: {
+            true: {
+                marginRight: '0',
+                paddingRight: '0',
+                overflowY: 'hidden',
+                overflowX: 'overlay',
+            },
+        },
+    },
+    defaultVariants: {
+        dark: false,
+        autohide: false,
+        has_horizontal: false,
+        is_only_horizontal: false,
+        is_only_horizontal_overlay: false,
+        is_scrollbar_hidden: false,
+    },
+});
+
+export type ScrollbarsProps = {
```

Comment:
```suggestion
export type ScrollbarsProps = {
```
Best to extend `HTMLAttributes<HTMLDivElement>` so that the `..props` can be typed when passing to `ScrollbarsContainer` (`ScrollbarsContainer` is also a `div` so it is a win-win here 😆 )